crates/rpc-types-eth/src/transaction/mod.rs

```diff
@@ -179,7 +180,7 @@ impl From<Transaction> for TxEnvelope {
     }
 }
 
-impl<T: TransactionTrait> TransactionTrait for Transaction<T> {
+impl<T: TransactionTrait + Typed2718> TransactionTrait for Transaction<T> {
```

Comment:
this should be redundant because guaranteed
```suggestion
impl<T: TransactionTrait> TransactionTrait for Transaction<T> {
```